dohproxy/client_protocol.py

```diff
@@ -25,6 +27,14 @@ def __init__(self, args, logger=None):
         if logger is None:
             self.logger = utils.configure_logger('StubServerProtocol')
         self.client = None
+        self.passthrough_domains = []
+        passthrough_args = getattr(self.args, 'passthrough_domains', '')
+        for domain in passthrough_args.split(','):
+            if not domain:
+                continue
+            if not domain.endswith('.'):
+                domain += '.'
+            self.passthrough_domains.append(dns.name.Name(domain.split('.')))
```

Comment:
you can use `argparse`'s `nargs=*` to directly support a list and change that to:

```
self.passthrough_domains = [dns.name.from_text(x) for x in self.args.passthrough_domains]
```